tests/conftest.py

```diff
@@ -14,7 +14,8 @@
 @pytest.fixture( params=[
         pytest.param("fortran", marks = pytest.mark.fortran),
         pytest.param("c", marks = pytest.mark.c),
-        pytest.param("python", marks = pytest.mark.python)
+        pytest.param("python", marks = pytest.mark.python),
+        pytest.param("cuda", marks = pytest.mark.cuda)
```

Comment:
Changing this here means that all tests are run via cuda by default. Do you think this is best? Or would it be better to create a second fixture which includes cuda and use it when test are capable of running with cuda?

Reply:
I think this should  be the better way to do it as now as seen before I needed to specify the file to not trigger all the tests.
but as i don't have any specific `Cuda` tests i am not sure on how to handle this.

Reply:
> I think this should be the better way to do it as now as seen before I needed to specify the file to not trigger all the tests.

Which of the 2 options mentioned is the _this_ that you think is the better way?

>but as i don't have any specific Cuda tests i am not sure on how to handle this.

I'm not sure what information you are missing as I'm not sure which suggestion you are talking about. My suggestion is to create a second feature e.g:
```python
@pytest.fixture( params=[
        pytest.param("fortran", marks = pytest.mark.fortran),
        pytest.param("c", marks = pytest.mark.c),
        pytest.param("python", marks = pytest.mark.python),
        pytest.param("cuda", marks = pytest.mark.cuda)
    ],
    scope = "session"
)
def language_with_cuda(request):
    return request.param
```
this would be used for tests which already work with cuda. Cuda-specific tests only need to run for cuda so they don't need to take a language argument

Reply:
Oh I see
but I have a question about the tests that I should activate in this PR.
as I want to just test a simple test as the ones in `epyccel/test_base.py`
can i just use `language_with_cuda` in the tests there that works in `Cuda` now
or just create a folder and put a simple test in it.

Reply:
Both would work. My suggestion was to create `language_with_cuda` and use it on the tests that you want to activate for all 4 languages. So for the tests in `epyccel/test_base.py`

Reply:
will start with `epyccel/test_base.py` first in the process of adding it on most of the tests eventual so we do not break the other tests.

Reply:
[e63992d](https://github.com/pyccel/pyccel-cuda/pull/32/commits/e63992d92d155aaa1bd03e5b49de5defcaa17d41)